platform-includes/performance/add-spans-example/native.mdx

```diff
@@ -37,3 +37,14 @@ void perform_checkout() {
 ```
 
 This example will send a transaction named `checkout` to Sentry. The transaction will contain a `validation` span that measures how long `validate_shopping_cart()` took and a `process` span that measures `process_shopping_cart()`. Finally, the call to `sentry_transaction_finish()` will finish the transaction and send it to Sentry.
+
+By default, transactions will inherit the `trace_id` and `parent_span_id` from the `propagation_context` as generated during SDK initialization. These values can be overridden by calling `sentry_set_trace()`, for example from a downstream SDK or to define a manual trace boundary:
+
+```c
+// Set the trace propagation_context which will be used
+//  for all events and transactions/spans
+const char *trace_id = "2674eb52d5874b13b560236d6c79ce8a";
+const char *parent_span_id = "a0f9fdf04f1a63df";
```

Comment:
Should we also provide an example of how to use `sentry_uuid_new_v4()` to generate these, or is that up to our users?

e.g. internally we use these helper functions to take what was generated & fit it into the appropriate-sized `sentry_value_t` containers
```c
    sentry_uuid_t trace_id = sentry_uuid_new_v4();
    sentry_uuid_t span_id = sentry_uuid_new_v4();
    sentry_value_set_by_key(sentry_value_get_by_key(*ctx, "trace"), "trace_id",
        sentry__value_new_internal_uuid(&trace_id));
    sentry_value_set_by_key(sentry_value_get_by_key(*ctx, "trace"), "span_id",
        sentry__value_new_span_uuid(&span_id));
```